R/drive_publish.R

```diff
@@ -53,9 +53,9 @@ build_drive_publish <-
     id <- file$id
 
     rev_id <- file$publish$revision
-    url <- file.path(.drive$base_url_files_v3, id, "revisions", rev_id)
+    endpoint <- file.path(id, "revisions", rev_id)
     build_request(
-      endpoint = url,
+      endpoint = endpoint,
```

Comment:
Again, this feels like some concatenation that can be handled via normal httr URL handling, if the info is stored in the expected place.